src/installer/managed/Microsoft.NET.HostModel/Bundle/TargetInfo.cs

```diff
@@ -37,21 +37,16 @@ public TargetInfo(OSPlatform? os, Architecture? arch, Version targetFrameworkVer
 
             Debug.Assert(IsLinux || IsOSX || IsWindows);
 
-            if (FrameworkVersion.CompareTo(net60) >= 0)
+            if (FrameworkVersion.CompareTo(net70) >= 0)
             {
-                BundleMajorVersion = 6u;
+                BundleMajorVersion = 7u;
                 DefaultOptions = BundleOptions.None;
             }
-            else if (FrameworkVersion.CompareTo(net50) >= 0)
+            else if (FrameworkVersion.CompareTo(net60) >= 0)
             {
-                BundleMajorVersion = 2u;
+                BundleMajorVersion = 6u;
                 DefaultOptions = BundleOptions.None;
             }
-            else if (FrameworkVersion.Major == 3 && (FrameworkVersion.Minor == 0 || FrameworkVersion.Minor == 1))
-            {
-                BundleMajorVersion = 1u;
-                DefaultOptions = BundleOptions.BundleAllContent;
-            }
```

Comment:
This is used by `BundleLegacy.cs`. So lets keep this case.

Reply:
Done.